src/components/CreateCategory.tsx

```diff
@@ -0,0 +1,74 @@
+// CreateCategory.tsx
+import React, { useState } from "react";
+import {Link } from "react-router-dom";
+import categoriesService from "../services/categories";
+import Alert from "./Alert"; 
+
+const CreateCategory = () => {  // Changed from React.FC to regular function declaration
+  const [name, setName] = useState("");
+  const [alertMessage, setAlertMessage] = useState(""); 
+  const [showAlert, setShowAlert] = useState(false); 
+
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    categoriesService
+      .create(name)
+      .then(() => {
+      })
```

Comment:
Why did you remove the redirect after success?

Reply:
Added it back